documentation/FAQ.md

```diff
@@ -1,6 +1,32 @@
 Frequently Asked Questions
 ==========================
 
+* On MacOS M1 if you see any of the following errors:
+    ```
+    (lldb) plugin load libsosplugin.so
+    error: this file does not represent a loadable dylib
+    ```
+    or if the sos plugin loads correct but you see this error on any command:
```

Comment:
change correct to correctly (I don't know why it wouldn't let me add a suggestion and change it myself

Reply:
That PR is closed, but I put in MacOS arm64, so I can switch it to just Mac